src/dockleHelper.ts

```diff
@@ -34,6 +37,58 @@ export async function getDockle(): Promise<string> {
     return dockleToolPath;
 }
 
+export function isCisChecksEnabled(): boolean {
+    return inputHelper.addCISChecks.toLowerCase() === "true";
+}
+
+export function getSummary(dockleStatus: number): string {
+    let summary = '';
+    switch(dockleStatus) {
+        case 0:
+            summary = 'No CIS benchmark violations were detected in the container image.'
+            break;
+        case DOCKLE_EXIT_CODE:
+            summary = getCisSummary();
+            break;
+        default:
+            summary = 'An error occured while scanning the container image for CIS benchmark violations.';
+            break;
+    }
+    
+    return `- ${summary}`;
+}
+
+export function getText(dockleStatus: number): string {
+    const cisIds = getCisIds(dockleStatus);
+    return `**Best Practices** -\n${cisIds.join('\n')}`;

```

Comment:
Why not `CIS Benchmarks` instead? 
What's the full form of CIS btw? :P

Reply:
Calling it `Best Practices Violations`